tests/bluetooth/tester/src/audio/btp_bap_broadcast.h

```diff
@@ -26,6 +26,7 @@ struct btp_bap_broadcast_stream {
  */
 struct btp_bap_broadcast_remote_source {
 	bt_addr_le_t address;
+	uint8_t source_id;
```

Comment:
Is this needed? Do we have multiple remote sources?

Reply:
Doesn't look like it, will remove